webapp/src/dogma/features/project/settings/mirrors/MirrorList.tsx

```diff
@@ -97,8 +99,20 @@ const MirrorList = <Data extends object>({ projectName }: MirrorListProps<Data>)
         },
         header: 'Status',
       }),
+      columnHelper.accessor((row: MirrorDto) => row.id, {
+        cell: (info) => (
+          <DeleteMirror
+            projectName={projectName}
+            id={info.getValue()}
+            deleteMirror={(projectName, id) => deleteMirror({ projectName, id }).unwrap()}
+            isLoading={isLoading}
+          />
+        ),
+        header: 'Actions',
```

Comment:
It is unusual that two `Actions` columns are in a row. How about moving `DeleteMirror` to the original `Actions` row and placing it next to `RunMirror`?
<img width="1232" alt="image" src="https://github.com/user-attachments/assets/9ae8d21e-e6e8-470d-895d-6093755297c1">

Otherwise, `<DeleteMirror>` could be exposed on the bottom of `MirrorView`. It may not be frequently used.

Reply:
> It is unusual that two Actions columns are in a row. 

Oops, I just check the last column which is `STATUS`. 😓 I've should implemented this before you implemented the run mirror. 🤣 
Let me add the dropdown component so that users can choose the actions. 

> <DeleteMirror> could be exposed on the bottom of MirrorView

That's something I want to do, but not in this PR. 😄 
I can't just add the button next to the editing button. Let me figure it out how to add it.

Reply:
<img width="907" alt="image" src="https://github.com/user-attachments/assets/55074847-5d1b-40b1-b5ff-5f5ffbcb8a43">
PTAL. 🙇

Reply:
@ikhoon Would you mind reviewing the changes? 😉